app/api/users/route.ts

```diff
@@ -0,0 +1,23 @@
+import { NextResponse } from "next/server";
+
+const neptune_url = "https://dzcmprdreb.execute-api.us-west-2.amazonaws.com/api/users"
+/**
+ * Creates a user given specific user attributes
+ */
+export async function POST(request: Request) {
+
+  var response = "Request Failed"
+
+  await request.json()
+  .then( async data => {
+
+    const res = await fetch(`${neptune_url}`, {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify(data)
+    });
+
+    response = await res.json();
+  })
+  return NextResponse.json({response})
+}
```

Comment:
You will need to change the Route handler to forward the status code that is returned from the API Gateway/Lambda to client or else NextResponse will by default send the `200` status code. 

See `api/post/route.ts` for an example.